pyomo/contrib/alternative_solutions/solution.py

```diff
@@ -0,0 +1,147 @@
+import json
+import pyomo.environ as pe
+from pyomo.common.collections import ComponentMap, ComponentSet
+from pyomo.contrib.alternative_solutions import aos_utils
+
+
+class Solution:
+    """
+    A class to store solutions from a Pyomo model.
+
+    Attributes
+    ----------
+    variables : ComponentMap
+        A map between Pyomo variables and their values for a solution.
+    fixed_vars : ComponentSet
+        The set of Pyomo variables that are fixed in a solution.
+    objectives : ComponentMap
```

Comment:
Nitpicking - should be `objective`?

Reply:
I don't think so.  A model may have more than one Objective component specified.

Reply:
The var in the code is `objective` - so something doesn't line up! ;)

Reply:
You're right.  The code isn't as general as I had thought.  {sigh}

I fixed the docs.